src/ScrollToTop.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
```

Comment:
Since we're just passing down children and not utilizing JSX, we don't actually need to import React! So, just do `import { Component } from 'react';`